CRM/MembersOnlyEvent/BAO/MembersOnlyEvent.php

```diff
@@ -75,7 +96,7 @@ public static function getMembersOnlyEvents($eventIDs) {
 
     $result = civicrm_api3('MembersOnlyEvent', 'get', [
       'sequential' => 1,
-      'is_groups_only' => 1,
+      'event_access_type' => 2,
```

Comment:
Why do we hardcode the type to two here? Since you introduce new access types, the value should be getting from method argument. Maybe something like this

```
public static function getMembersOnlyEvents($eventIDs, int $eventAccessType) {
....
      'event_access_type' => eventAccessType,
...
}
```

Reply:
Good catch!.
Created the commit `WHF-308: Refactor the method getMembersOnlyEvents to use event access type`.